policy_packs/azure/network/check_virtual_networks_have_flowlogs_enabled/policies.tf

```diff
@@ -0,0 +1,77 @@
+# Azure > Network > Virtual Network > Approved
+resource "turbot_policy_setting" "azure_network_virtual_network_approved" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/azure-network#/policy/types/virtualNetworkApproved"
+  value    = "Check: Approved"
+}
+
+# Azure > Network > Virtual Network > Approved > Usage
+resource "turbot_policy_setting" "azure_network_virtual_network_approved_usage" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/azure-network#/policy/types/virtualNetworkApprovedUsage"
+  value    = "Approved"
+}
+
+# Azure > Network > Virtual Network > Approved > Custom
+resource "turbot_policy_setting" "azure_network_virtual_network_approved_custom" {
+  resource       = turbot_policy_pack.main.id
+  type           = "tmod:@turbot/azure-network#/policy/types/virtualNetworkApprovedCustom"
+  template_input = <<-EOT
+    - |
+      {
+        resource {
+          name:get(path:"name")
+          flowLogs:get(path:"flowLogs")
+        }
+      }
+    - |
+      {
+        resource {
+          name:get(path:"name")
+          flowLogs:get(path:"flowLogs")
+        }
+        resources(filter:"resourceTypeId:tmod:@turbot/azure-networkwatcher#/resource/types/flowLog $.id:'{{ $.resource.flowLogs[0].id }}'"){
+          items {
+            enabled:get(path:"enabled")
+            id:get(path:"id")
+            name:get(path:"name")
+          }
+        }
+      }
+    EOT
+  template       = <<-EOT
+    {%- set virtualNetworkFlowLogs = $.resource.flowLogs %}
+    {%- set subscriptionFlowLogs = $.resources.items %}
+
+    {%- if virtualNetworkFlowLogs and virtualNetworkFlowLogs | length > 0 %}
+      {%- for vnetFlowLog in virtualNetworkFlowLogs %}
+        {%- set matchedFlowLog = null %}
+        {%- for subscriptionFlowLog in subscriptionFlowLogs %}
+          {%- if subscriptionFlowLog.id == vnetFlowLog.id %}
+            {%- set matchedFlowLog = subscriptionFlowLog %}
+          {%- endif %}
+        {%- endfor %}
+        
```

Comment:
Please use the existing format for setting approved > custom policies. For reference please check: https://github.com/turbot/guardrails-samples/blob/main/policy_packs/aws/ec2/enforce_instances_use_amis_with_approved_tags/policies.tf#L13